stubs/extensions/pdo.phpstub

```diff
@@ -150,16 +146,6 @@ class PDOStatement implements Traversable
      * @return false|T
      */
     public function fetchObject($class = \stdclass::class, array $ctorArgs = array()) {}
-
-    /**
-     * @psalm-taint-sink sql $value
-     */
-    public function bindValue(string|int $param, mixed $value, int $type = PDO::PARAM_STR): bool {}
```

Comment:
Don't remove the methods please, just remove the taint sink annotation.

Reply:
Done !